specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2023-03-01-preview/mongoCluster.json

```diff
@@ -0,0 +1,983 @@
+        },
+        "nextLink": {
+          "description": "The link used to get the next page of results.",
+          "type": "string",
+          "readOnly": true
+        }
+      },
+      "description": "A list of mongo clusters."
+    },
+    "MongoCluster": {
+      "type": "object",
+      "description": "Represents a mongo cluster resource.",
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/TrackedResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/MongoClusterProperties",
+          "x-ms-client-flatten": true,
+          "description": "Properties of the mongo cluster."
+        }
+      }
+    },
+    "MongoClusterUpdate": {
+      "type": "object",
+      "description": "Represents a mongo cluster resource for updates.",
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/MongoClusterProperties",
+          "x-ms-client-flatten": true,
+          "description": "Properties of the mongo cluster."
+        },
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Application-specific metadata in the form of key-value pairs."
+        }
+      }
+    },
+    "MongoClusterProperties": {
+      "type": "object",
+      "description": "The properties of a mongo cluster.",
+      "properties": {
+        "createMode": {
+          "type": "string",
+          "description": "The mode to create a mongo cluster.",
+          "enum": [
+            "Default",
+            "PointInTimeRestore"
+          ],
+          "x-ms-enum": {
+            "name": "CreateMode",
+            "modelAsString": true
+          },
+          "x-ms-mutability": [
+            "create"
+          ],
+          "default": "Default"
+        },
+        "restoreParameters": {
+          "$ref": "#/definitions/MongoClusterRestoreParameters",
+          "description": "Parameters used for restore operations",
+          "x-ms-mutability": [
+            "create"
+          ]
+        },
+        "administratorLogin": {
+          "type": "string",
+          "description": "The administrator's login for the mongo cluster.",
+          "x-ms-mutability": [
+            "create",
+            "update",
+            "read"
+          ]
+        },
+        "administratorLoginPassword": {
+          "type": "string",
+          "x-ms-secret": true,
+          "format": "password",
+          "x-ms-mutability": [
+            "create",
+            "update"
+          ],
+          "description": "The password of the administrator login."
+        },
+        "serverVersion": {
+          "type": "string",
+          "description": "The Mongo DB server version. Defaults to the latest available version if not specified."
+        },
+        "connectionString": {
```

Comment:
So does this property contain secrets or not?

Reply:
No secrets in connection string, it contains necessary properties to connect aside from username and password which aer explicitly placeholders.